exampleSite/config.toml

```diff
@@ -20,6 +20,7 @@ preserveTaxonomyNames = true
   customCSS = ["css/custom.css"]
   customJS = ["js/custom.js"]
   hideSocialMenu = false
+  hideReadTime = false
```

Comment:
Could you please change this option to `showReadingTime` with default value to `false`?
The current default behavior will be preserved that way.